src/test/java/com/kaltura/client/test/tests/servicesTests/bookmarkTests/ListTests.java

```diff
@@ -0,0 +1,86 @@
+package com.kaltura.client.test.tests.servicesTests.bookmarkTests;
+
+import com.kaltura.client.enums.AssetType;
+import com.kaltura.client.enums.BookmarkActionType;
+import com.kaltura.client.enums.BookmarkOrderBy;
+import com.kaltura.client.test.servicesImpl.BookmarkServiceImpl;
+import com.kaltura.client.test.tests.BaseTest;
+import com.kaltura.client.test.utils.BookmarkUtils;
+import com.kaltura.client.types.Bookmark;
+import com.kaltura.client.types.BookmarkFilter;
+import com.kaltura.client.types.ListResponse;
+import com.kaltura.client.utils.response.base.Response;
+import io.qameta.allure.Description;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ListTests extends BaseTest {
+
+    private String assetId = "606283";
```

Comment:
if that is media id you have to use IngestVOD and then take data from ingested item.